### PR TITLE
fix a typo

### PR DESCRIPTION
**Pull request by luojie233**
Signed-off-by: Jie Luo <luo612@zju.edu.cn>

depencences -> dependences

### CODE REVIEW

File: community/developer-summit-2016/application_service_definition_notes.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Is this different from what the PAASes already do?  It's not that different, we
 
 Being able to view an application as a single unifying concept is a major desire.   Want to click "my app" and see all of the objects associated with it.  It would be an overlay on top of Kubernetes, not something in core.
 
-One pending feature is that you can't look up different types of controllers in the API, that's going to be fixed.  Another one is that we can't trace the depenences; helm doesn't label all of the components deployed with the app.
+One pending feature is that you can't look up different types of controllers in the API, that's going to be fixed.  Another one is that we can't trace the dependences; helm doesn't label all of the components deployed with the app.
```
Comment (jonboulle) [inline community/developer-summit-2016/application_service_definition_notes.md]: dependencies

**Comment by jonboulle** [comment]
lgtm, though you should squash them together

**Comment by luojie233** [comment]
Yes, THANKS!

**Comment by luojie233** [comment]
I merge it.

**Comment by pigmej** [comment]
lgtm thx!

**Pull request merged**